tests/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ if (ENABLE_GTEST)
     endforeach()
 
     set(language_tests
+        mesh_generation.cpp
```

Comment:
Not sure where to put this but language tests seems odd.

Reply:
Maybe a new group called `utility_tests`?